examples/angular_bazel_architect/projects/frontend-lib/src/lib/frontend-lib.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import {ComponentFixture, TestBed, waitForAsync} from '@angular/core/testing';
```

Comment:
Change came from a "yarn format" which was requested by a pre-commit hook when I merged in the latest stable branch. Can revert if needed